Python/Product/Debugger/Debugger/IPythonDebugOptionsService.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABILITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+namespace Microsoft.PythonTools.Debugger {
+    public interface IPythonDebugOptionsService {
+        /// <summary>
+        /// True to ask the user whether to run when their code contains errors.
+        /// Default is false.
+        /// </summary>
+        bool PromptBeforeRunningWithBuildError { get; }
+        /// <summary>
```

Comment:
For readability, you should add a blank line between properties (before the docs for the next property)
```
/// ...
bool yyy { get; }

/// ...
bool zzz {get; }
```